### ISSUE TITLE
Is there any drop down method and mouse actions method in winapp driver as it is in selenium?

### ISSUE THREAD

**Issue by shubhamwaykar** [microsoft/WinAppDriver#1316]

**Comment by naeemakram** [comment]
No, there's no separate method for drop down/combo box. The drop down control is made up of a list of items and a down arrow button with name "Open". 
When you click the open button, the list of items is displayed on screen. You can pick what you want by iterating through that list. 
You can also use the SendKeys method to select an item from a drop down in some cases. Hope this helps.

**Comment by Wolfe1** [comment]
@shubhamwaykar what language are you looking to do this in? If its Robot Framework we have keywords set up for both:

- [Select Element From ComboBox](https://accruent.github.io/robotframework-zoomba/DesktopLibraryDocumentation.html#Select%20Element%20From%20ComboBox)
- [Various Mouse keywords](https://accruent.github.io/robotframework-zoomba/DesktopLibraryDocumentation.html#Mouse%20Over%20Element)

[There is a full sample file here](https://github.com/Accruent/robotframework-zoomba/blob/master/samples/DesktopTests.robot)

Otherwise if its python you can take a look at the source code to get an idea of how to control those functions: https://github.com/Accruent/robotframework-zoomba/blob/master/src/Zoomba/DesktopLibrary.py

**Comment by shubhamwaykar** [comment]
@Wolfe1 we are using node.js
